vpn.py

```diff
@@ -59,8 +63,19 @@ def scan_vpns(directory, auth_file, exclude_list):
     vpn_dir = return_abs_path(directory, "vpns")
     conf_dir = return_abs_path(directory, "configs")
     auth_file = return_abs_path(".", auth_file)
-    for filename in os.listdir(conf_dir):
-        logging.info("Moving onto %s" % (filename))
+    conf_list = os.listdir(conf_dir)
+
+    if shuffle_lists:
+        shuffle(conf_list)
```

Comment:
Though there is no guaranteed order, the order of directory listing on the same machine is always the same. 
The option to randomize the order was added mostly for testing and is useful when testing a lengthy experiment on random VPs.